.circleci/config.yml

```diff
@@ -5,9 +5,10 @@ jobs:
   build:
     environment:
       _JAVA_OPTIONS: "-Xms512m -Xmx512m"
+      GRADLE_OPTS: "-Dorg.gradle.daemon=false -Dorg.gradle.workers.max=1"
```

Comment:
I suspect that with jdk 11, we may need something like https://stackoverflow.com/a/56939344 -- i.e. rather than relying on Xms and Xmx